src/app/frontend/common/components/middleellipsis/component.js

```diff
@@ -19,10 +19,26 @@ export default class MiddleEllipsisController {
   /**
    * Constructs middle ellipsis controller.
    * @ngInject
+   * @param {!angular.JQLite} $element
    */
-  constructor() {
+  constructor($element) {
     /** @export {string} Initialized from the scope. */
     this.displayString;
+
+    /** @private {!angular.JQLite} */
+    this.element_ = $element;
+  }
+
+  isTrancated() {
+    let c = this.element_;
+    c.addClass("kd-middleellipsis_width");
```

Comment:
Use Angular's `ng-class`.

Reply:
I'm sorry, but I don't understand you.
What should I do?

Reply:
https://docs.angularjs.org/api/ng/directive/ngClass

Reply:
I have no idea how to use `ng-class` effectively.
The example of the link is how to add class dynamically, but I use addClass method because of adding class temporarily.
Please help me.